README.md

```diff
@@ -13,18 +13,18 @@ Start using it straight away. (Following example assumes that it was installed v
 ```php
 <?php
 
-require 'path/to/vendor/autoload.php';
+require __DIR__.'/vendor/autoload.php';
```

Comment:
Since this is just a README therefore the path as set to generic/pseudo type. I would recommend reverting this.